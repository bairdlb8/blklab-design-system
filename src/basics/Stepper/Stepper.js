import React from 'react'
import PropTypes from 'prop-types'

import './Stepper.scss'

const Stepper = ({className=''}) => {
  return <div>Stepper</div>
}

Stepper.propTypes = {}

export default Stepper
