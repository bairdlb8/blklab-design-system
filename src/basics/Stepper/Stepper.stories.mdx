import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import Stepper from './Stepper'

<Meta title="Basics / Stepper" component={ Stepper } />

# Stepper

## Add some text here

export const Template = (args) => {
  return <Stepper {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ Stepper } />
