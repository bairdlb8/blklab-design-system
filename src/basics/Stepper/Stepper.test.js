import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { toHaveNoViolations } from 'jest-axe'
import Stepper from './Stepper'

expect.extend(toHaveNoViolations)

describe('<Stepper />', () => {

  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Stepper />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Stepper />)

    expect(asFragment(<Stepper />)).toMatchSnapshot()
  })

})
