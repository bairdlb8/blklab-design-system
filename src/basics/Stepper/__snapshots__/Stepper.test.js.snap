// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stepper /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Stepper
  </div>
</DocumentFragment>
`;
